titles/rel-notes-rhdh/title-rhdh-release-notes.adoc

```diff
@@ -14,8 +14,24 @@ include::artifacts/snip-customer-support-info.adoc[]
 
 //includes
 include::modules/release-notes/con-release-notes-overview.adoc[leveloffset=+1]
-include::modules/release-notes/con-relnotes-notable-features.adoc[leveloffset=+1]
-include::modules/release-notes/con-breaking-changes.adoc[leveloffset=+1]
-include::modules/release-notes/con-relnotes-techpreview-features.adoc[leveloffset=+1]
-include::modules/release-notes/con-relnotes-known-issues.adoc[leveloffset=+1]
-include::modules/release-notes/con-relnotes-fixed-issues.adoc[leveloffset=+1]
+
+
+include::assemblies/assembly-release-notes-new-features.adoc[leveloffset=+1]
+
+
+include::assemblies/assembly-release-notes-breaking-changes.adoc[leveloffset=+1]
+
+
+include::assemblies/assembly-release-notes-deprecated-functionalities.adoc[leveloffset=+1]
+
+
+include::assemblies/assembly-release-notes-technology-preview.adoc[leveloffset=+1]
+
+
+include::assemblies/assembly-release-notes-fixed-security-issues.adoc[leveloffset=+1]
```

Comment:
security now listed above bug fixes?  I'm OK with that but are you sure we want to order things that way? 

In 1.2 we list fixes then CVEs. https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.2/html-single/release_notes_for_red_hat_developer_hub_1.2/index#fixed-issues-in-red-hat-developer-hub-1-2-2